sdk/python/feast/infra/online_stores/contrib/mysql_online_store/util.py

```diff
@@ -0,0 +1,325 @@
+from __future__ import absolute_import
```

Comment:
@franciscojavierarceo 
Majority of functionality lives here. I'll need to think through each of these functions to ensure we're rolling back and handling row locks/unlocks when necessary.